src/components/map/draw-tools/DrawingToolbox.tsx

```diff
@@ -0,0 +1,135 @@
+import { useState } from 'react';
+
+import { Feature, Map } from 'ol';
+import { GeoJSON } from 'ol/format';
+import { Circle, Geometry } from 'ol/geom';
+import { Type } from 'ol/geom/Geometry';
+import { fromCircle } from 'ol/geom/Polygon';
+import { Draw, Interaction, Modify, Snap } from 'ol/interaction';
+import { defaults } from 'ol/interaction/defaults';
+import { Options, createBox } from 'ol/interaction/Draw';
+import VectorSource from 'ol/source/Vector';
+import { PiCircle, PiPolygonFill, PiRectangle } from 'react-icons/pi';
+
+import { DATA_PROJECTION, MAP_PROJECTION } from '@/components/Map';
+import { useFilters } from '@/hooks/useFilters';
+
+import { DrawingTool } from './DrawingTool';
+
+import './DrawingToolbox.scss';
+
+type DrawingToolboxProps = {
+  isToolboxVisible: boolean;
+  map: Map;
+  drawingSource: VectorSource<Feature<Geometry>>;
+};
+
+enum Shapes {
+  CIRCLE = 'Circle',
```

Comment:
I'm not sure if we should support Circle, it's an unusual drawtool to use on a map due to how many points it can end up creating. We can leave it in and during testing we can see how well it works.

Reply:
I disagree, I've used it in almost every map I've built over the last 15 years. Radius tools are especially useful. However, as you say, if they don't like it, it's easy to remove.